### ISSUE TITLE
System Shock Remake Ultra-Wide Patch possible?

### ISSUE THREAD

**Issue by Hoshi82** [RoseTheFlower/UltrawideIndex#42]
It's not an issue and more asking if this would be possible? The devs screwed it up again and having a propper aspect ratio in 21:9 would be super nice. :)

**Comment by RoseTheFlower** [comment]
Hey, I had a fix built but tested only on the demo, then Lyall [released](https://cdn.discordapp.com/attachments/1113233781256638556/1113370074716327977/SystemShock_SUWSF_v2.zip) something very similar, so I don't think there would be value in releasing what I have.

**Comment by Hoshi82** [comment]
Thanks! This works like a charm.

**Comment by Hoshi82** [comment]
Seems the Github page from Lyall is gone. Is the fix available somwhere else?

**Comment by RoseTheFlower** [comment]
Yeah, Lyall's account got suspended over a ToS violation unknown to us (and perhaps even Lyall), but the patches are now [available on Codeberg](https://codeberg.org/Lyall/UltrawidePatches/src/branch/main/System%20Shock%20%282023%29).

**Comment by Hoshi82** [comment]
Thanks! :)
